raiden/ui/cli.py

```diff
@@ -48,13 +49,13 @@
     click.option(
         '--registry-contract-address',
         help='hex encoded address of the registry contract.',
-        default='32c5dab9b099a5b6c0e626c1862c07b30f58d76a',  # testnet default
+        default='bbc60aa23059b039407ac008bd0b7e902890d382',  # testnet default
         type=str,
     ),
     click.option(
         '--discovery-contract-address',
         help='hex encoded address of the discovery contract.',
-        default='79ab17cc105e820368e695dfa547604651d02cbb',  # testnet default
+        default='524b7dcacac3055bd42fc03b006e9fdcb607e2be',  # testnet default
```

Comment:
What about defining these constants in the constant module? including the older addresses